### DISCUSSION TITLE
外部 demo 模块使用 ant-design/charts 失败

### DISCUSSION THREAD

**Discussion by changhuaixin** [umijs/dumi#2122] [Q&A] [answered]
错误信息 
![image](https://github.com/umijs/dumi/assets/61184708/0f63c6bd-4b69-47b0-8572-6fd6af44dd95)

复现步骤：
1. npx create-dumi 创建一个新的项目
2. 在 guide.md 引入 ant-design/charts 画的折线图
3. 访问 http://localhost:8000/guide

guide.md：
![image](https://github.com/umijs/dumi/assets/61184708/b4a8deee-95ba-4025-a070-1ef8bb74c42d)

line.tsx
![image](https://github.com/umijs/dumi/assets/61184708/c8bcd583-67ac-4ab5-95fd-ef6f989365ee)

**Comment by changhuaixin** [comment]
package.json:
{
  "name": "tproject",
  "version": "0.0.1",
  "description": "A static site based on dumi",
  "scripts": {
    "start": "npm run dev",
    "dev": "dumi dev",
    "build": "dumi build",
    "preview": "dumi preview",
    "prepare": "husky install && dumi setup"
  },
  "authors": [
    "huaixin.chx@alibaba-inc.com"
  ],
  "license": "MIT",
  "commitlint": {
    "extends": [
      "@commitlint/config-conventional"
    ]
  },
  "lint-staged": {
    "*.{md,json}": [
      "prettier --write --no-error-on-unmatched-pattern"
    ]
  },
  "devDependencies": {
    "@commitlint/cli": "^17.1.2",
    "@commitlint/config-conventional": "^17.1.0",
    "dumi": "^2.3.0",
    "husky": "^8.0.1",
    "lint-staged": "^13.0.3",
    "prettier": "^2.7.1"
  },
  "dependencies": {
    "@ant-design/charts": "^2.1.1",
    "@antv/f-react": "^1.3.1",
    "@antv/f2": "^5.4.5",
    "antd": "^5.17.3",
    "d3": "^7.9.0"
  }
}

**Comment by PeachScript** [comment, marked as answer]
配置 `mfsu: false` 试试
